examples/house-credit-default/main.py

```diff
@@ -38,140 +46,153 @@
 
 
 def export_to_pb(sess, x, filename):
-  pred_names = ['output']
-  tf.identity(x, name=pred_names[0])
+    pred_names = ["output"]
+    tf.identity(x, name=pred_names[0])
 
-  graph = graph_util.convert_variables_to_constants(
-      sess, sess.graph.as_graph_def(), pred_names)
+    graph = graph_util.convert_variables_to_constants(
+        sess, sess.graph.as_graph_def(), pred_names
+    )
 
-  graph = graph_util.remove_training_nodes(graph)
-  path = graph_io.write_graph(graph, ".", filename, as_text=False)
-  print('saved the frozen graph (ready for inference) at: ', path)
+    graph = graph_util.remove_training_nodes(graph)
+    path = graph_io.write_graph(graph, ".", filename, as_text=False)
+    print("saved the frozen graph (ready for inference) at: ", path)
 
 
 def print_nodes(graph):
-  """Print a list of nodes from a tf.Graph."""
-  print([n.name for n in graph.as_graph_def().node])
+    """Print a list of nodes from a tf.Graph."""
+    print([n.name for n in graph.as_graph_def().node])
 
 
 def build_model(input_shape):
-  """Build a logistic regression model with tf.keras."""
-  model = keras.Sequential([
-      layers.Dense(1, use_bias=False, activation='sigmoid',
-                   input_shape=[input_shape]),
-  ])
+    """Build a logistic regression model with tf.keras."""
+    model = keras.Sequential(
+        [
+            layers.Dense(
+                1, use_bias=False, activation="sigmoid", input_shape=[input_shape]
```

Comment:
it's times like these that black is a little excessive :D

Reply:
indeed :)